containers/daap-python-base/src/var/task/data_product_metadata.py

```diff
@@ -0,0 +1,117 @@
+import json
+import os
+import traceback
+
+import boto3
+import botocore
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+from data_platform_logging import DataPlatformLogger
+from dataengineeringutils3.s3 import get_filepaths_from_s3_folder, read_json_from_s3
+
+s3_client = boto3.client("s3")
+
+
+# These can be moved to separate module for all paths when work starts on that
+def get_bucket_name():
+    return os.environ["BUCKET_NAME"]
+
+
+def get_data_product_metadata_path(data_product_name):
```

Comment:
Question: do we want all functions/classes in these shared modules to have type annotations? I don't have an opinion one way or the other, just wanted to check before I start contributing to this myself.

Reply:
actually types are there. Maybe you were looking at an earlier commit?

Reply:
Ah yeah, I went through the earlier commits first, ignore that then 🙂

Reply:
I've been using os pathing, we should standardise our approach